teuthology/openstack/setup-openstack.sh

```diff
@@ -347,7 +347,7 @@ function setup_dnsmasq() {
 
 function subnet_names_and_ips() {
     local subnet=$1
-    python -c 'import netaddr; print "\n".join([str(i) for i in netaddr.IPNetwork("'$subnet'")])' |
+    python -c 'import netaddr; print("\n".join([str(i) for i in netaddr.IPNetwork("'$subnet'")]))' |
```

Comment:
can we assume python version is greater or equal to 2.6 here?

Reply:
> Python 2.6 is not supported anymore

thanks. i didn't realize this. just to make sure we are on the same page. where did you get this message? by checking the "classifiers" section in `setup.py` or by checking `tox.ini` ?

> Why are you asking?

because python2.5 does not support `print()` at all.

Reply:
do we have systems with python2.5 where teuthology is used?

Reply:
Regarding python2.5 
https://www.python.org/downloads/release/python-255/
> Release Date: Jan. 31, 2010
> This is a source-only release that only includes security fixes. The last full bug-fix release of Python 2.5 was Python 2.5.4. User are encouraged to upgrade to the latest release of Python 2.7 (which is 2.7.2 at this point).

It was almost 10 years ago. :-D

Reply:
Regarding python2.6
https://www.python.org/downloads/release/python-269/
> Python 2.6.9
> Release Date: Oct. 29, 2013
> With the 2.6.9 release, and five years after its first release, the Python 2.6 series is now officially retired. All official maintenance for Python 2.6, including security patches, has ended. For ongoing maintenance releases, please see the Python 2.7 series. The NEWS file lists every change in each alpha, beta, release candidate, and final release of Python 2.6.

They said 'retired' 6 years ago.

Reply:
> do we have systems with python2.5 where teuthology is used?

that's my question.

Reply:
If there are such systems (which I doubt), the question is: do we have any reason to support them?

Reply:
```kyr@teuthology:~$ python2 --version
Python 2.7.12
```
This is what we have on teuthology server in sepia.